app/src/test/java/com/uc/mymovies/MovieUnitTest.kt

```diff
@@ -12,7 +12,7 @@ import org.junit.Assert.*
  */
 class MovieUnitTest {
     @Test
-    fun movieDTO() {
+    fun movieDTO_matainsGenre() {
```

Comment:
Ah yes.  Good to put description in unit test function name, so you know what is failing.